src/js/chapter-factory.js

```diff
@@ -0,0 +1,87 @@
+import { makeDraggable, makeDroppable } from "./drag-drop";
+
+//Chapter factory
+const Chapter = (story, images, question, chapterNumber) => {
```

Comment:
Imo: put Chapter into an extra file. Also have a look what I posted into Discord if the additional properties make sense to you.

Reply:
Sure, will move to separate file.